AmplifyPlugins/DataStore/Tests/DataStoreHostApp/DataStoreStressTests/README.md

```diff
@@ -0,0 +1,250 @@
+## DataStore Stress Tests
+
+The following steps demonstrate how to set up DataStore with a conflict resolution enabled API through amplify CLI, with API key authentication mode. 
+
+
+### Set-up
+
+1. `amplify init` and choose `iOS` for type of app you are building
+
+2. `amplify add api`
+
+```perl
+? Please select from one of the below mentioned services: `GraphQL`
+? Provide API name: `apiName`
+? Choose the default authorization type for the API `API key`
+? Enter a description for the API key:
+? After how many days from now the API key should expire (1-365): `365`
+? Do you want to configure advanced settings for the GraphQL API `Yes, I want to make some additional changes.`
+? Configure additional auth types? `No`
+? Configure conflict detection? `Yes`
+? Select the default resolution strategy `Auto Merge`
+? Choose a schema template: `Blank Schema`
+? Do you want to edit the schema now? Y
+```
+When asked to provide the schema, create the `schema.graphql` file
```

Comment:
is this the same schema as Android stress tests? If not, there will be more work in the future to align the tests cases.

Reply:
Currently this schema is different than android.

Reply:
looks like the schema has been greatly reduced to just one model, which means it's probably not that necessary to reuse the same schema, as Android stress tests could simply do the same paralleism on any model regardless of the name.